shell/platform/darwin/macos/framework/Source/AccessibilityBridgeMacDelegate.mm

```diff
@@ -0,0 +1,318 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#import "flutter/shell/platform/darwin/macos/framework/Source/AccessibilityBridgeMacDelegate.h"
+
+#import "flutter/shell/platform/embedder/embedder.h"
+
+#import "flutter/shell/platform/darwin/macos/framework/Source/FlutterPlatformNodeDelegateMac.h"
+
+namespace flutter {  // namespace
+
+bool SystemVersionEqualTo(NSOperatingSystemVersion target) {
```

Comment:
These three methods seem like they should probably be in a commonly accessible target.